src/shared/common/CalendarHeatmap.js

```diff
@@ -63,87 +135,93 @@ function CalendarHeatmap(props: Props) {
     const itemWidth = Math.min((screenWidth - SPACING_MEDIUM * 2 - 1) / 7, 38);
 
     const now = new Date();
+    now.setHours(0);
+    now.setMinutes(0);
+    now.setSeconds(0);
+    now.setMilliseconds(0);
+
```

Comment:
Since this is repeated most of the time. Let's use the golden rule of reusability and create a function for this. This just looks ugly appearing at lots of places.